oshmem/mca/sshmem/mmap/sshmem_mmap_module.c

```diff
@@ -126,6 +126,7 @@ segment_create(map_segment_t *ds_buf,
     /* init the contents of map_segment_t */
     shmem_ds_reset(ds_buf);
 
+    (void)munmap(mca_sshmem_base_start_address, size);
```

Comment:
probably not needed

Reply:
We now "reserve" that area by holding an mmap() on it as it seems there is no randomization between mmap/munmap + mmap sequence and area could be consumed by unrelated mmap() in between.

 Then on the modules we "overwrite" it with (ucp_mem_map() / mmap() / shmat()). It's a try to make it explicit, although it opens for race and mmap() anyways replaces it with `MAP_FIXED`.

Will remove, need to check with shmat() that it overwrites existing area too.

Reply:
removed for mmap module, kept for sysv module as it is needed